src/quo2/components/wallet/account_overview/component_spec.cljs

```diff
@@ -0,0 +1,67 @@
+(ns quo2.components.wallet.account-overview.component-spec
+  (:require [quo2.components.wallet.account-overview.view :as account-overview]
+            [test-helpers.component :as h]
+            [utils.i18n :as i18n]))
+
+(h/describe
+ "Account overview test"
+ (h/test "renders correct account name"
+         (h/render [account-overview/view
+                    {:account-name "Diamond Hand"}])
+         (h/is-truthy (h/get-by-text "Diamond Hand"))
+
+         (h/test "renders correct account value"
+                 (h/render [account-overview/view
+                            {:current-value "20.0$"}])
+                 (h/is-truthy (h/get-by-text "20.0$")))
+
+         (h/test "renders correct account changes"
+                 (h/render [account-overview/view
+                            {:currency-change   "€0.00"
+                             :percentage-change "0.00%"}])
+                 (and
+                  (h/is-truthy (h/get-by-text "0.00%"))
+                  (h/is-truthy (h/get-by-text "€0.00"))))
+
+         (h/test "renders correct timeframe"
+                 (h/render [account-overview/view
+                            {:time-frame-string    "15 May"
+                             :time-frame-to-string "19 May"}])
+                 (and
+                  (h/is-truthy (h/get-by-text "15 May"))
+                  (h/is-truthy (h/get-by-text "19 May"))))
+
+         (h/test "renders correct account timeframe"
+                 (h/render [account-overview/view
+                            {:time-frame        :one-week
+                             :currency-change   "€0.00"
+                             :percentage-change "0.00%"}])
+                 (h/is-truthy (h/get-by-text (i18n/label :t/one-week-int))))
```

Comment:
oh I believe we don't want to use `i18n` directly in tests - 🤔 instead we just use the key for the translation string - please see:
https://github.com/status-im/status-mobile/blob/develop/src/quo2/components/dividers/strength_divider/component_spec.cljs#L12